app/src/main/java/com/codesoom/assignment/common/exception/TaskNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.common.exception;
+
+
+/**
+ * 찾는 Task가 없을 경우 TaskNotFoundException 예외를 정의한다.
+ */
+public class TaskNotFoundException extends RuntimeException {
```

Comment:
이렇게 해 보세요.
```suggestion
/**
 * 작업을 찾지 못했을 때 던집니다.
 */
public class TaskNotFoundException extends RuntimeException {
```
다음은 `IllegalArgumentException` 클래스에 있는 JavaDoc 설명입니다. 이걸 읽어보고 곰곰히 생각해 보세요. 그리고 `throw` 키워드의 의미에 대해서도 생각해 보세요.
![image](https://user-images.githubusercontent.com/1855714/161382690-55ccb29b-120d-458c-9c01-2642081e9307.png)

Reply:
넵!! 감사합니다